server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -608,8 +609,18 @@ private void run() {
 
       Span waLogs = Trace.start("walogs");
       try {
+        RemovalPolicy removalPolicy = RemovalPolicy.DELETE;
+        if (shouldArchiveFiles()) {
+          if (isUsingTrash()) {
+            removalPolicy = RemovalPolicy.ARCHIVE_TRASH;
+          } else {
+            removalPolicy = RemovalPolicy.ARCHIVE;
```

Comment:
Could make this more readable with: `removalPolicy = isUsingTrash() ? RemovalPolicy.ARCHIVE_TRASH : RemovalPolicy.ARCHIVE;`